torch/csrc/jit/ir.h

```diff
@@ -110,6 +111,8 @@ struct Scope {
   Scope* parent_;
   Symbol name_;
   std::vector<std::unique_ptr<Scope> > children_;
+  std::map<std::string, int> constants_;
```

Comment:
Are you very sure you want sorted map? `std::map` is a heavy, sorted RB tree. We always prefer `std::unordered_map` (hashtable) if the sorting is not required